StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/BottomSheetViewController.swift

```diff
@@ -392,7 +394,35 @@ class BottomSheetViewController: UIViewController, BottomSheetPresentable {
         hideKeyboardGesture.delegate = self
         view.addGestureRecognizer(hideKeyboardGesture)
     }
-
+    #if compiler(>=6.2)
+    func enableNavigationBarBlurInteraction() {
+        guard #available(iOS 26.0, *),
+              navigationBarContainerView.interactions.first(where: { $0 is UIScrollEdgeElementContainerInteraction }) == nil else {
```

Comment:
instead of this, could you see if `navigationBarBlur.view == nil` 

```
[var view: UIView?](https://developer.apple.com/documentation/uikit/uiinteraction/view)
The view that owns the interaction.
Required
```